Update 26.1 (#272)
DevExpress/dotnet-eud: eud-for-devexpress-reports/reporting-for-desktop/report-designer/report-designer-for-winforms/preview-print-and-export-reports.md

Reviewer: ```suggestion
To switch a report to print preview mode, click the **Preview** tab. You will see your report populated with data and broken down into pages, as specified.
```

Reviewer: ```suggestion
> To learn more about the options available in print preview mode, refer to [Print Preview for WinForms](../../print-preview/print-preview-for-winforms.md) section of this documentation.
```

Reviewer: ```suggestion
When in Print Preview mode, you can print out your report using appropriate menu and toolbar commands.
```

Reviewer: ```suggestion
When in Print Preview mode, you can export your report to files in different formats. The resulting files can either be saved to the hard drive or sent by e-mail.
```

Reviewer: What table? Spacify.

Author: ```suggestion
The table below describes the following:
```

Reviewer: ```suggestion
You can specify how a Table should be treated by screen readers in the exported PDF document. To do this, set the control's **Accessible Role** property to **Table** before you export a report.
```

Reviewer: ```suggestion
If an image watermark conveys meaning and you want to include it in the PDF document logical structure, set **Role** to **Figure** when you create or edit the watermark:
```

Reviewer: ```suggestion
Use the **Accessible Role** property to specify how screen readers treat **Picture Box**, **Shape**, **Bar Code**, and **Zip Code** controls in exported PDF documents. You can change the value to **Decorative** or keep the default value of **Figure**.
```

## Export a Report to PDF with Accessible Tags (PDF/UA Compatibility)

> Use the **Accessible Role** option to specify how report elements should be treated by screen readers in the exported PDF document. Set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report to PDF format.

Reviewer: ```suggestion
Use the **Accessible Role** option to specify how report elements should be treated by screen readers in the exported PDF document. Set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report as a PDF.
```

| `Watermark` (an image watermark) | Treated as an artifact; excluded from the PDF logical structure. | Figure | 
| `Watermark` (a text watermark) | Treated as an artifact; excluded from the PDF logical structure. | Paragraph |
| `Picture Box`, `Shape`, `Bar Code`, `Zip Code` | Treated as a `Figure`. | Decorative (Artifact) | 
| `Page Info` | Treated as an artifact; excluded from the PDF logical structure. | Paragraph (use for meaningful content such as dates or page numbers) |
| `Panel` | Included in the document structure. | Decorative (Artifact) (use for layout-only or purely visual panels) |

The **Accessible Description** property is not in effect for artifacts.

> [!NOTE]
> **Rich Text** content is automatically converted to semantic tags when exported to a tagged PDF: headings become `H1`–`H3`, paragraphs become `P`, lists become `L`/`LI`, images become `Figure`, and tables become `Table`/`TR`/`TH`/`TD`. No role configuration is needed.

### Define Label Accessible Role

Set the control’s **Accessible Role** property to **Heading 1 - Heading 6** before you export a report.

![Set AccessibleRole in Property Grid](../../images/label-assesible-role.png)

> In the PDF Export Options dialog, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report to PDF format.

Reviewer: ```suggestion
In the PDF Export Options dialog, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report as a PDF.
```

![Set AccessibleRole in Property Grid](../../images/table-assesible-role.png)

> In the PDF Export Options dialog, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report to PDF format.

Reviewer: ```suggestion
In the PDF Export Options dialog, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report as a PDF.
```

> The image below shows the result. **Accessible Role** is set to **Table**, and the screen reader treats Table as a table in the exported document:

Reviewer: ```suggestion
The image below shows the result. **Accessible Role** is set to **Table**, and the screen reader treats that Table as a table in the exported document:
```

![Set AccessibleRole in Property Grid](../../images/table-row-assesible-role.png)

> In the PDF Export Options dialog, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report to PDF format.

Reviewer: ```suggestion
In the PDF Export Options dialog, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to PDF/UA specification. Then, export the report as a PDF.
```

> [!NOTE]
> **Accessible Role** is not in effect for cells merged with the Cell's **Row Span** property. 

> In the PDF Export Options dialog, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report to PDF format.

Reviewer: ```suggestion
In the PDF Export Options dialog, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to PDF/UA specification. Then, export the report as a PDF.
```

> The image below shows the result. **Table Cell**'s b is set to **Table Header Cell**, and the screen reader treats Table Cell with "Bill to:" text as a header cell in the exported document:

Reviewer: Clarify which image? This image or the image below, etc.

Author: The sentence below says `The image below ` - I am not sure what I need to clarify

![Image Accessible Role Property Set to Figure](../../images/image-watermark-accessible-role.png)

Before you export your report, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to make the document PDF/UA compatible.

> The image below shows the result. **Role** is set to **Figure**, and the screen reader treats watermark as a figure in the exported document:

Reviewer: ```suggestion
The image below shows the result. **Role** is set to **Figure**, and the screen reader treats the watermark as a figure in the exported document:
```

![Text Accessible Role Property Set to Paragraph](../../images/text-watermark-accessible-role.png)

Before you export your report, set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to make the document PDF/UA compatible.

> The image below shows the result. **Role** is set to **Paragraph**, and the screen reader treats watermark as a paragraph in the exported document:

Reviewer: ```suggestion
The image below shows the result. **Role** is set to **Paragraph**, and the screen reader treats the watermark as a paragraph in the exported document:
```

> [!NOTE]

> > Do not exclude elements that carry meaning, use this tag only for decorative elements.

Reviewer: ```suggestion
> Do not exclude elements that carry meaning; use this tag only for decorative elements.
```

### Digital Signature Accessible Description

You can specify an accessible description for the **PDF Signature** control to help screen readers identify its content.

- **Document Signature**
   If the control displays a document signature, use the **Accessible Description** option in the control's **PDF Signature Options**:

    ![Document signature Accessible Description property](../../images/document-digital-signature-description.png)

- **Signature Placeholder**
   If the control is a signature placeholder (the document signature is disabled), use the control's **Accessible Description** property:

    ![Signature placeholder Accessible Description property](../../images/xrpdfsignature-description.png)

The following image shows both signatures in the PDF tag tree:

![Digital signatures in the PDF tag tree](../../images/digital-signature-in-a-tag-tree.png)

### Limitations

* The **Never Embedded Fonts** export option is not supported for PDF/UA-compatible documents.

> * If a report contains an **XRPdfContent** control, you cannot export the report to PDF with the PDF/UA option enabled.

Reviewer 2: ```suggestion
* If a report contains a **PDF Content** control, you cannot export the report to PDF with the PDF/UA option enabled.
```
